examples/src/main/java/com/mapbox/navigation/examples/core/IndependentRouteGenerationActivity.kt

```diff
@@ -294,6 +294,7 @@ class IndependentRouteGenerationActivity : AppCompatActivity() {
 
     override fun onDestroy() {
         super.onDestroy()
+        mapboxReplayer.finish()
```

Comment:
Do developers now need to remember doing this as part of their `onDestroy`? If so, we should add a warning note on the docs.

Reply:
Do they? Could you clarify?

Reply:
At least `stop` definitely has to be called, otherwise `ReplayEventSimulator` will continue simulation after `Activity` finish. `finish` looks like a function, where we could add more cleanup, if necessary, so the user should call it.